### DISCUSSION TITLE
Syslog solution

### DISCUSSION THREAD

**Discussion by mdepedrof** [tteck/Proxmox#1328] [Ideas]
Would be great if can create a srcipt to deploy a LXC with a syslog solution. the BEST solution would be ELK but I think its very difficult. Maybe Graylog would be an acceptable option to have a centralized log system.

whe you start to create multiples LXC an systems on your network, its very difficult to stay informed abaout all logs and events that occurs on your environments. This is because if you implement a central log system, you can send all logs of the all of your systems there and then, you can do querys, alerts, etc...

**Comment by phrozen77** [comment]
Or the recently-released/open-sourced "Supabase Logs": https://supabase.com/blog/supabase-logs-self-hosted
